apps/docs/src/app/core/component-docs/carousel/examples/carousel-loading-content.component.html

```diff
@@ -0,0 +1,17 @@
+<div style="display: flex; justify-content: center;">
+    <fd-busy-indicator [loading]="loading">
```

Comment:
Shouldn't this be integrated into the carousel?

Reply:
Done. have moved it inside carousel item.